content/windows/docs/networking.md

```diff
@@ -71,6 +71,30 @@ DSR enablement can be verified following the instructions in the [Microsoft Netw
 
 ![](./images/dsr.png)
 
+If preserving your available IPv4 addresses and minimizing wastage is crucial for your subnet, it is generally
+recommended to avoid using prefix delegation mode as mentioned in
+[Prefix Mode for Windows - When to avoid](https://aws.github.io/aws-eks-best-practices/networking/prefix-mode/index_windows/#avoid-prefix-delegation-when).
+If using prefix delegation is still desired, you can take steps to optimize IPv4 address utilization in your subnet. The
+documentation Prefix Mode for 
+[Windows - Configure parameters](https://aws.github.io/aws-eks-best-practices/networking/prefix-mode/index_windows/#configure-parameters-for-prefix-delegation-to-conserve-ipv4-addresses)
```

Comment:
right now, the link style is pointing out to the internet

will you change the link style to reference the files in the repo?

I sent you a patch with an example.

Reply:
@geoffcline Please take a look. The URLs have been updated as suggested.